kernel-specification.yml

```diff
@@ -4689,6 +4734,54 @@ kernels:
     automatic-tests: true
     manual-tests: []
 
+  - name: awkward_UnionArray_mergetags_const
+    specializations:
+      - name: awkward_UnionArray_mergetags_to8_const
+        args:
+          - {name: totags, type: "List[int8_t]", dir: out}
+          - {name: toindex, type: "List[int64_t]", dir: out}
+          - {name: fromleft, type: "Const[List[int64_t]]", dir: in, role: IndexedArray-index}
+          - {name: leftlen, type: "int64_t", dir: in, role: default}
+          - {name: fromright, type: "Const[List[int64_t]]", dir: in, role: IndexedArray-index}
+          - {name: rightlen, type: "int64_t", dir: in, role: default}
+    description: null
+    definition: |
+      def awkward_UnionArray_mergetags_const(totags, toindex, fromleft, leftlen, fromright, rightlen):
+            ileft = 0
+            iright = 0
+            leftsize = 0
+            rightsize = 0
+            tag = 0
+            j = 0
+            li = 0
+            ri = 0
+            length = leftlen + rightlen
+            for m in range(length):
+              if tag == 0 and ileft < leftlen:
+                leftsize = fromleft[ileft + 1] - fromleft[ileft]
+                for i in range(leftsize):
+                  toindex[j] = li + 1
+                  li = li + 1
+                  totags[j] = tag
+                  j = j + 1
+              if tag == 1 and iright < rightlen:
+                rightsize = fromright[iright + 1] - fromright[iright]
+                for i in range(rightsize):
+                  toindex[j] = ri + 1
+                  ri = ri + 1
+                  totags[j] = tag
+                  j = j + 1
+              if tag == 0:
+                if iright < rightlen:
+                  tag = 1
+                ileft = ileft + 1
+              else:
+                if ileft < leftlen:
+                  tag = 0
+                iright = iright + 1
```

Comment:
Same thing: 4-space indent. (Your first indentation is 6 spaces.)

Reply:
done